pandas/plotting/_core.py

```diff
@@ -919,6 +929,105 @@ def _make_plot(self):
             ax.errorbar(data[x].values, data[y].values,
                         linestyle='none', **err_kwds)
 
+    def _get_plot_bubbles(self, size_data):
```

Comment:
For ease of testing, I think this should be a static method and take `(size_data, n_bubble_points=200, size_factor=1)`. I don't care whether it's a static method or a standalone function in the module.

Reply:
OK. I guess it makes sense to also change the `_get_legend_bubbles` function to a static method with the same signature (`size_data, n_bubble_points=200, size_factor=1`)?